kolibri/plugins/coach/assets/src/composables/useQuizCreation.js

```diff
@@ -70,7 +70,7 @@ export default function useQuizCreation(DEBUG = false) {
       console.error("You're trying to generate test data in production. Please set DEBUG = false.");
     }
     // First let's make some QuizQuestion objects so we have them to initialize resources with
-    // Typically this data would be fetched and usable from the useQuizResources module
+    // Typically this data would be fetched and usable from the useExerciseResources module
```

Comment:
what is the difference between a quiz exercise (i.e. `useExerciseResources` and a `QuizExercise`). Is it because the `useExerciseResources` module will be shared and repurposed beyond "quizzes"?

Reply:
maybe this is actually now a commit that got overwritten, but there is still a reference to `useExerciseResources` in the new diff.

Reply:
Yeah the `useExerciseResources` module is what I'm calling the "navigating the tree, checkbox logic, assessmentmetadata annotations, etc" module. 

I think that it will be used in future updates to Lesson as well (or at least want it to be an option for that).